README.md

```diff
@@ -1,4 +1,47 @@
+## Summary
+
+The Managed Velero Operator is used for backups in OpenShift Dedicated v4. It is based on an open source project [Velero]( https://velero.io) that uses a controller model where it monitors custom resources and takes actions based on their states. The Managed Velero Operator dynamically creates and configures prerequisites for Velero and it then deploys the Velero software on the OpenShift Dedicated cluster. Velero backs up the Kubernetes object store: all the deployments, pods, config maps, secrets, etc. In addition to Kubernetes configuration pieces, Velero also handles backing up and any persistent volumes that are attached to the cluster. For OpenShift Dedicated v4, these are AWS EBS volumes.
+
+## What the Managed Velero Operator Does
+
+1. When the Managed Velero Operator starts, it checks whether it is installed on a supported platform. For OpenShift Dedicated v4 environment it validates that 
+	+ it is installed on AWS
+	+ it has been installed with installer provisioned infrastructure
+	+ it has all the needed details in the cluster's infrastructure configuration to provision Velero
+
+2. Next, the Managed Velero Operator begins the **Reconcile loop**. It checks whether the Velero custom resources are created/installed and it ensures that they are created/installed before it takes any further action.
+
+3. Next, the Managed Velero Operator starts up the manager and controller and waits for the initial configuration.
+
+4. Once the Managed Velero Operator detects its custom resources and understands that it's ready to provision Velero in this cluster, it will check if an S3 bucket already has been provisioned to store the Kubernetes object store, the metadata and the details of the backup. If the Managed Velero Operator detects that there isn't an S3 bucket defined in its custom resources, it will provision an S3 bucket for that use. **The Managed Velero Operator will ensure the following settings when the bucket is created and it will check periodically to ensure that the bucket settings are re-enforced.**
+	+ the bucket is encrypted
+	+ the public access to the bucket is enforced turned off
+	+ the permissions and the life cycle settings on the bucket are correct
+
+5. Next, the Managed Velero Operator will configure and install the Velero software. This includes ensuring that setup manifests are installed, and Velero custom resources such as the volume storage location and the backup storage location are specified. This step also provisions credentials for Velero to access the AWS bucket through the cluster credentials operator and a credentials request custom resource that is part of OpenShift v4.
+
+6. Finally, the Managed Velero Operator completes the **Reconcile loop**. 
+
+The Managed Velero Operator will listen to changes in settings and custom resources and periodically run the Reconcile loop to change the settings back to what it expects.
+
+## Requirements
+
++ Access to a Kubernetes cluster version 1.7 or later.
```

Comment:
OpenShift 4.1+ (this won't work with vanilla kubernetes, as we retrieve cloud provider information from the infrastrctures.config.openshift.io object)